tex/chapters/electric-field/infinite-plane.tex

```diff
@@ -0,0 +1,19 @@
+\section{Electric field generated by an infinite plane}
+Let $\sigma$ be the electric charge density per unit of area of an infinite plane, located on the $XY$-plane of a coordinate system $(x,y,z)$. In accordance with \ref{eq:compact-electric-field}, the infinitesimal contribution to the electric filed given by an infinitesimal area $\total{A}=\total{x}\total{y}$ is
+\be
+\total{\mathbf{E}(\mathbf{r})}=\frac{\sigma}{4\pi\epsilon_0}\frac{\total{x}\total{y}}{r^{2}}\uvec{r}
+\ee
+At a point $\mathbf{r}_{0}=(x_{0},y_{0},z_{0})$, the infinitesimal area $dxdy$ located at $(x,y,0)$ can be expressed as $(\rho\cos(\theta)+x_{0},\rho\sin(\theta)-y_{0},0)$, so that, by changing $\theta$, a circle centered at $(x_{0},y_{0},0)$ is described. The vector $\mathbf{r}$ becomes $(-\rho\cos(\theta),-\rho\sin(\theta),z_{0})$.
```

Comment:
it should be `+y_{0}` instead of `-y_{0}`

Reply:
Changed.